mackup/applications/sublime-text-3.cfg

```diff
@@ -2,6 +2,7 @@
 name = Sublime Text 3
 
 [configuration_files]
-Library/Application Support/Sublime Text 3/Packages/User
```

Comment:
That's what is live https://github.com/lra/mackup/blob/master/mackup/applications/sublime-text-3.cfg
So I should close this PR?